.github/workflows/daily-workflow-ci.yml

```
@@ -21,13 +21,19 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: '3.2'
+      - name: Set report name
+        run: |
+          BRANCH=${{ github.head_ref }}
+          REPORT_NAME="${BRANCH/\//_}"
```

Why do you need to create `BRANCH` only to create `REPORT_NAME`? I think this should work

```suggestion
          REPORT_NAME="${{ github.head_ref //\//_ }}"
```

---

We tried this and other suggestions and none of them seem to work properly with substitution syntax.